ww4ff/settings.py

```diff
@@ -24,7 +24,7 @@ class TrainingSettings(BaseSettings):
     device: str = 'cuda:0'
     batch_size: int = 16
     lr_decay: float = 0.75
-    max_window_size_seconds: float = 1
+    max_window_size_seconds: float = 0.75
```

Comment:
what was this change for `max_window_size_seconds`?

Reply:
I was testing out a different default setting. We still use 1 second (see envs/res8.env).